taskcluster/kinds/corpus-clean-parallel-bicleaner-ai/kind.yml

```diff
@@ -104,7 +105,10 @@ tasks:
                     pip install $MOZ_FETCHES_DIR/kenlm-0.0.0-cp310-cp310-linux_x86_64.whl &&
                     pip install -r {bicleaner_reqs} &&
                     export PYTHONPATH=$PYTHONPATH:$VCS_PATH &&
-                    export PATH=$PATH:~/.local/bin &&
+                    export CUDA_DIR="$MOZ_FETCHES_DIR/cuda-toolkit" &&
+                    export PATH="$PATH:~/.local/bin:$CUDA_DIR/bin" &&
+                    export XLA_FLAGS="--xla_gpu_cuda_data_dir=$CUDA_DIR" &&
```

Comment:
Is it worth moving to `bicleaner.sh` as something too implementation-specific? I don't know, maybe it's fine to have it here.